.travis.yml

```diff
@@ -480,8 +480,12 @@ script:
   # Configure yarp
   - if $TRAVIS_WITH_INTEGRATION_TESTS; then build/bin/yarp conf 0 0 local; fi
 
-  # Run unit tests
-  - if $TRAVIS_RUN_UNIT_TESTS; then (cd build; travis_wait 50 ctest --output-on-failure --build . -C ${TRAVIS_BUILD_TYPE}); fi
+  # Run unit tests (but do not fail on error)
+  - if $TRAVIS_RUN_UNIT_TESTS; then (cd build; travis_wait 50 ctest --output-on-failure --build . -C ${TRAVIS_BUILD_TYPE} || /bin/true); fi
```

Comment:
Please note that macOS has no `/bin/true`, but `/usr/bin/true` 😄.
We can either symlink `/usr/bin/true` in `/bin/true` when a macOS build happen or add a condition for this command.

Reply:
Or we can just drop support for macOS, that makes a lot of sense to me! :rofl: 
Just kidding, perhaps just `|| true`

Reply:
Ahahaha! That would probably work 🙂!